crt_portal/cts_forms/forms.py

```diff
@@ -856,6 +856,26 @@ def clean(self):
             return cleaned_data
 
 
+class Profiles(ModelForm):
```

Comment:
Good opportunity here to move towards more conventional Django naming practices for modelForms, let's call this class `ProfileForm`

```suggestion
class ProfileForm(ModelForm):
```

https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/#modelform